core/templates/domain/classroom/classroom-backend-api.service.ts

```diff
@@ -45,7 +45,7 @@ interface ClassroomDataBackendDict {
   providedIn: 'root'
 })
 export class ClassroomBackendApiService {
-  classroomData: ClassroomData = null;
+  classroomData!: ClassroomData;
```

Comment:
why can this never be `undefined` or `null`?

Reply:
classroomData comes from classroom-data.model.ts file which never returns null. The default value isn't being used anywhere so I removed it. I've also covered to the model file to be extra sure.

Reply:
Please add a comment about this.

Reply:
done